recipes/qt5/meta.yaml

```diff
@@ -41,12 +52,12 @@ requirements:
     - curl  # [win]
     - flex  # [linux]
     - fontconfig 2.11.*  # [linux]
-    - freetype  2.5* # [linux]
+    - freetype  2.5*  # [linux]
     - gst-plugins-base  # [linux]
-    - icu 56.*
```

Comment:
Yes, the problem is not the recipe, the problem is I don't have time to go through creating new packages for ICU and Boost.